app/admin/requested_volunteers.rb

```diff
@@ -15,6 +15,7 @@ def create
 
     def update
       super do |success, failure|
+        notify_volunteers_updated if success.present? && should_notify_update?
         success.html { redirect_to admin_organisation_request_path(resource.request_id) }
```

Comment:
Why not make it part of the success block? 

```suggestion
        success.html do
          notify_volunteers_updated if should_notify_update?
          redirect_to admin_organisation_request_path(resource.request_id)
        end
```

Reply:
That's a good point. thanks!